WooCommerce/Classes/ViewRelated/MainTabBarController.swift

```diff
@@ -375,8 +379,10 @@ private extension MainTabBarController {
             let productsTabIndex = WooTab.products.visibleIndex(isHubMenuFeatureFlagOn)
             controllers.insert(productsNavigationController, at: productsTabIndex)
 
-            let reviewsTabIndex = WooTab.reviews.visibleIndex(isHubMenuFeatureFlagOn)
-            controllers.insert(reviewsNavigationController, at: reviewsTabIndex)
+            if !ServiceLocator.featureFlagService.isFeatureFlagEnabled(.hubMenu) {
+                let reviewsTabIndex = WooTab.reviews.visibleIndex(isHubMenuFeatureFlagOn)
+                controllers.insert(reviewsNavigationController, at: reviewsTabIndex)
+            }
```

Comment:
nit: similar to above, could these setup be in the `else` case below?
also, could this just use the private property `isHubMenuFeatureFlagOn` instead of calling `ServiceLocator.featureFlagService.isFeatureFlagEnabled(.hubMenu)`?
```suggestion
            if !isHubMenuFeatureFlagOn {
                let reviewsTabIndex = WooTab.reviews.visibleIndex(isHubMenuFeatureFlagOn)
                controllers.insert(reviewsNavigationController, at: reviewsTabIndex)
            }
```